templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/statement_detail_subheader.html.twig

```diff
@@ -39,7 +39,7 @@
 {% endif %}
 
 
-<div class="u-pt-0_125 layout--flush">
+<div class="u-pt-0_125 layout--flush flex">
```

Comment:
could you try to remove the `flow-root` class in line 102 instead of adding flex here? i guess that introduced the issue in the first place (shame on me).

Reply:
thank you, this is indeed more elegant than adding another class. done here https://github.com/demos-europe/demosplan-core/pull/2356/commits/ad17afe38a7475976616044d763b5b5ea6bd5dd4